### ISSUE TITLE
Sleep/Wake issues

### ISSUE THREAD

**Issue by Doyle37** [Mc-Colour/HP-Elitebook-840-G7-Hackintosh#1]
Have you fixed the Sleep/Wake issue?

**Comment by Mc-Colour** [comment]
No, I no longer have access to the machine so I can't work on it anymore. All I will say is it sleep did work if you used the machine docked, like a desktop. Good luck
